Runigram.java

```diff
@@ -134,6 +213,16 @@ public static Color[][] blend(Color[][] image1, Color[][] image2, double alpha)
 	 */
 	public static void morph(Color[][] source, Color[][] target, int n) {
 		//// Replace this comment with your code
+		if (source.length != target.length || source[0].length != target[0].length ){
+			target = scaled(target, source[0].length, source.length);
+		}
+		for (int i = 0; i <= n; i++){
+			double alpha = (n - i) / n;

```

Comment:
incorrect n and i are integers. 4/5 = 0. You should cast one of the variables to double 4.0/5 = 0.8
(n-i) / (double) n